listener/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(listener)
+
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+  set(CMAKE_CXX_STANDARD_REQUIRED ON)
+endif()
```

Comment:
If C++14 is always required to use RxCPP, we may want to export that requirement from `rxcpp` and `rxros` so downstream packages don't all need to add these 4 lines to their `CMakeLists.txt`.

Reply:
@henrik7264 should we? (I think you are right @gavanderhoorn that this should be done a level above, but Henrik should confirm first).